converter/internal/promtailconvert/testdata/docker_relabel.river

```diff
@@ -14,10 +14,12 @@ discovery.relabel "flog_scrape" {
 }
 
 loki.source.docker "flog_scrape" {
-	host             = "unix:///var/run/docker.sock"
-	targets          = discovery.docker.flog_scrape.targets
-	forward_to       = [loki.write.default.receiver]
-	relabel_rules    = discovery.relabel.flog_scrape.rules
+	host          = "unix:///var/run/docker.sock"
+	targets       = discovery.docker.flog_scrape.targets
+	forward_to    = [loki.write.default.receiver]
+	relabel_rules = discovery.relabel.flog_scrape.rules
+
+	http_client_config { }
```

Comment:
This is a little unfortunate, but not a blocker; can you spend like 15 minutes on this to see if we can stop it from generating this?